tests/cpp.test

```diff
@@ -277,6 +277,7 @@
 30845  ben_026.cfg                          cpp/deref.cpp
 
 30850  sp_cmt_cpp_start_force.cfg           cpp/sp_cmt_cpp_start.cc
+30852  Issue_2138.cfg                       cpp/Issue_2138.cpp
```

Comment:
Fine for the input file, but for single-option configs, I like naming them e.g. `sp_paren_qualifier-f.cfg`.